website/blog/2016-03-11-blog-post.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Blog Title
```

Comment:
Why is this boilerplate in here?

Reply:
https://github.com/Microsoft/fast-dna/pull/1175#discussion_r237646930